libpyvinyl/BaseFormat.py

```diff
@@ -4,7 +4,10 @@
 
 
 class BaseFormat(AbstractBaseClass):
-    """The abstract format class. It's the interface of a certain data format."""
```

Comment:
The abstract format class which serves as the common interface for derived format classes.

Reply:
Done.